.github/workflows/workflow.yaml

```diff
@@ -113,3 +113,46 @@ jobs:
         run: make test.e2e
         env:
           CGO_ENABLED: "0"
+
+  load-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        qps: [1,10,100]
+        payload-size: [256, 8192]
+        duration: [30]
+        target-example: ["http_headers"]
+        image: [
+          "envoyproxy/envoy-dev:latest",
+        ]
+    name: Load Test (${{ matrix.image }}, ${{ matrix.target-example }}, ${{ matrix.qps }} QPS, ${{ matrix.payload-size }} KB, ${{ matrix.duration }} s)
+    needs: [build-examples]
```

Comment:
**I said** please write more representative extensions for load tests. We need to load under various  realistic usecases.